### PR TITLE
Scan results in directory API

### LINKED ISSUE #734: Add landing page scan results to API
In #527 we added a `manage.py` command that exports the landing page scan results in CSV format, which is handy for quickly determining which SecureDrops have significant landing page issues (e.g., use of Google Analytics). This command is run as a cron job and the result is published to http://securedrop.org/media/documents/scanresults.csv (we've never announced that URL, but it is referenced e.g. in #532). 

This is adding to the technical complexity of SecureDrop.org, and we may temporarily lose this CSV as part of the migration to k8s. If possible, it would be good to make the latest scan result available via the directory API, as we do for SecureTheNews.

### PR DESCRIPTION
**Pull request by chigby**
This pull request adds a `latest_scan` field to entries in the directory API. This field contains the complete information for the most recent scan result for that directory. I've also included some minor updates to some related test factories.

To test this, you can load the url http://localhost:8000/api/v1/directory/ in a browser or with a tool like `curl`. 

image

Fixes #734

### CODE REVIEW

File: directory/api/serializers.py [hunk 1]
```diff
@@ -38,4 +39,25 @@ class Meta:
...
+            'latest_scan',
         ]
+
+    def get_latest_scan(self, directory_entry):
+        latest = directory_entry.get_live_result()
+
+        if latest:
+            serializer_latest = ScanResultSerializer(instance=latest)
+            return serializer_latest.data
+        else:
```
Comment (SaptakS) [inline directory/api/serializers.py]: Can skip the `else` since return statement.

**Review by SaptakS — COMMENTED** [review]
In general, looks good to me. Left a small comment. The tests for API look good!

File: directory/api/serializers.py (hunk 1 above)
Comment (chigby) [reply directory/api/serializers.py]: Good point here, I've removed it.

**Review by SaptakS — APPROVED** [review]
Looks good to me

**Pull request merged**